docs/monitoring/infrastructure.md

```diff
@@ -20,9 +20,7 @@ Sematext Cloud will automatically create one Infra App named "Default Infra App"
 
 A Golang-based [Sematext Agent](../agents/sematext-agent) will be installed on your machine. This Agent sends all infrastructure data to an Infra App whose token is configured during the Agent's installation.
 
-The Infra App also includes data about your containerized environments. Installation instructions for Docker Monitoring also include an Infra App token. That way when installing Sematext Agent for Docker, an Infra App will be installed as well.
-
-Infra Apps are currently available to all users for free as a Beta feature.
+The Infra App also includes data about your containerized environments. Installation instructions for container Monitoring also include an Infra App token. That way when installing Sematext Agent for Docker, an Infra App will be installed as well.
```

Comment:
No more "Sematext Agent for Docker".  Maybe this needs to be updated?

Reply:
I understood it as installing STA to monitor docker env. I'll reword a bit.